code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -223,6 +223,7 @@
 
 	var/obj/item/device/radio/R = new /obj/item/device/radio/headset/syndicate/alt(agent)
 	R.set_frequency(radio_freq)
+	R.name = "strange headset"
```

Comment:
Isn't literally half the station "alien" in some sense? Or is this a new level of racism/speciesism in the future? "Yeah, we got humans, cat people, dog/fox people, lizard men, robots, people made out of the shit we're here to mine who burst into flames if they touch oxygen, bug people, but no this thing here is totally 'alien'."